tf_files/cloud.tf

```diff
@@ -20,6 +20,17 @@ resource "aws_vpc" "main" {
     }
 }
 
+data "aws_vpc_endpoint_service" "s3" {
+    service = "s3"
+}
+
+resource "aws_vpc_endpoint" "private-s3" {
```

Comment:
Do you mean all the `aws_route_table_association`s? Based on the docs I thought that would create a problem. Can the ids just be added to the [ `route_table_ids` list](https://github.com/uc-cdis/cloud-automation/pull/73/files#diff-383a6efad1c0b20acd60b88df7870d79R31)?